api/features/versioning/managers.py

```diff
@@ -53,5 +54,11 @@ def _get_latest_versions(
 
         return self.raw(
             get_latest_versions_sql,
-            params={"environment_id": environment_id, "api_key": environment_api_key},
+            params={
+                "environment_id": environment_id,
+                "api_key": environment_api_key,
+                # It seems as though there is a timezone issue when using postgres's
+                # built in now() function, so we pass in the current time from python.
+                "live_from_before": timezone.now().isoformat(),
```

Comment:
I would prefer avoiding the use of Python's now() here. Maybe we can leave a TODO here to revisit it in the future?

Reply:
I've updated the comment to use a TODO.

Reply:
Yeah, so this works (after updating the test to schedule the other version for 2 hours later): 

```sql
and efv2."live_from" <= now() + interval '90 minutes'
```

Indicating that it likely is some TZ issue here.

Reply:
Actually, that's not true because even this works: 

```sql
and efv2."live_from" <= now() + interval '1 second'
```